src/main/java/net/neoforged/neoforge/common/extensions/IItemExtension.java

```diff
@@ -580,15 +581,15 @@ default boolean canDisableShield(ItemStack stack, ItemStack shield, LivingEntity
 
     /**
      * @return the fuel burn time for this item stack in a furnace. Return 0 to make
-     *         it not act as a fuel. Call super to let the {@link NeoForgeDataMaps#FURNACE_FUELS} decide.
-     * @apiNote This method takes precedence over the {@link NeoForgeDataMaps#FURNACE_FUELS data map}.
-     *          However, you should use the data map unless necessary (i.e. NBT-based burn times) so that users can configure burn times.
-     * @implNote Overriders should call super instead of directly returning -1.
+     *         it not act as a fuel. Return -1 to let the default vanilla logic
+     *         decide.
+     * @apiNote This method takes precedence over the {@link net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps#FURNACE_FUELS data map}. * However, you should use the data map unless necessary (i.e. NBT-based burn times) so that users can configure burn times.
```

Comment:
You seem to have merged a separate line into this line accidentally -- the portion after the second asterisk (`*`).

Reply:
The portion after the asterisk is still here. It should be put on its own line (or the asterisk should be removed).
```suggestion
     * @apiNote This method takes precedence over the {@link net.neoforged.neoforge.registries.datamaps.builtin.NeoForgeDataMaps#FURNACE_FUELS data map}. 
     *          However, you should use the data map unless necessary (i.e. NBT-based burn times) so that users can configure burn times.
```